### PR TITLE
Sydney updates and refactoring

### PR DESCRIPTION
**Pull request by josh-**

### CODE REVIEW

File: sydney/index.html [hunk 1]
```diff
@@ -5,26 +5,75 @@
...
+  <div class="google-maps" id="syd-2018-map"></div>
+</div>
+
+<div class="page-block event-tweet event-quote no-border">
+  <div class="unihack-wrap container-fluid">
+    <div class="row justify-content-md-center">
+      <div class="col-md-12">
+        <i class="icon fa fa-twitter"></i>
+        <p class="tweet">
+          Absolutely love hearing all the ideas at <a href="https://twitter.com/unihacksyd/">@unihacksyd</a> !! Excited to hear who the winners are 🎉
```
Comment (erfanio) [inline sydney/index.html]: is the space `</a> !!` intentional?

Comment (josh-) [reply sydney/index.html]: Ahh it is! From the tweet: https://twitter.com/j_ennychu/status/899130527256412160

**Comment by terencehuynh** [comment]
Looks good @josh-. Merge it in!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: sydney/index.html (hunk 1 above)
Comment (erfanio) [reply sydney/index.html]: Sorry I didn't realise it's a quote

**Pull request merged**